release/artifacts.yaml

```diff
@@ -39,10 +39,10 @@
   type: jar-scala
 - target: //daml-lf/interpreter:interpreter
   type: jar-scala
-- target: //daml-lf/language:language
-  type: jar-scala
 - target: //daml-lf/kv-transaction-support:kv-transaction-support
   type: jar-scala
+- target: //daml-lf/language:language
```

Comment:
why did you reshuffle this?

Reply:
I had a merge conflict and moved it manually with the proper alphabetical order :)